include/seqan3/std/ranges

```diff
@@ -30,6 +30,22 @@ template<class T>
     requires ::std::derived_from<T, ::std::ranges::view_base>
 //!\endcond
 inline constexpr bool enable_view<T> = true;
+
+// std::ranges::borrowed_range's are valid range-v3 borrowed_range's
+//!\cond
+template <::std::input_or_output_iterator I, ::std::sentinel_for<I> S, ::std::ranges::subrange_kind K>
```

Comment:
doesn't the `::std::sentinel_for<I>` trigger the visibility bug?

Reply:
no, because this path will only be used with gcc>=10.